.vscode/launch.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Any reason we should add this file? I'm fine doing so within the company's code, but does it really make sense in an external library?

Reply:
I dont know, I like it there.  I don't personally see a difference between internal and external.  I think we're better off when we develop the same way no matter where the code is hosted.

Reply:
If someone submits a pull request with files from a different IDE, do we also want to check those in as well?

Reply:
I don't have a problem with that, within reason.  If you're using an IDE that adds 100 files to the artifact, then no, your IDE is stupid. 

I'm coming from a perspective of - lets make this as easy to contribute to for the people who are contributing to it rather than worry about contributors and IDEs we don't actually have.

If we decide its a bad idea, we can always delete files - won't even fail a build.

Reply:
If you don't think that's a convincing argument (and I can think of reasons it may not be), we should discuss how we work on Belt too or why this is substantially different.

Reply:
Can we at least get rid of "Attach"?

Reply:
Definitely.